lively.server/plugins/test-runner.js

```diff
@@ -9,7 +9,7 @@ export default class TestRunner {
     let results;
     try {
     this.headlessSession = new HeadlessSession();
-    await this.headlessSession.open('http://localhost:9011/worlds/load?name=__newWorld__&askForWorldName=false',  (sess) => sess.runEval(`typeof $world !== 'undefined' && $world.name == 'aLivelyWorld' && $world._uiInitialized`));
+    await this.headlessSession.open('http://localhost:9011/worlds/load?name=__newWorld__&askForWorldName=false',  (sess) => sess.runEval(`typeof $world !== 'undefined' && $world?.name == 'aLivelyWorld' && $world._uiInitialized`))
```

Comment:
Is this an accident? In which cases will this kick in? We do have the `undefined` check before...

Reply:
Good question, I did this in response to the test runner starting to crash on me. I did not investigate it further. Maybe it can go now?

Reply:
I removed it and it works for me, I'll go ahead and change this back.